mlCompared.html

```diff
@@ -326,25 +326,15 @@
 let myFuncs = {
   myFun = (fun x -> x + 1);
   your = (fun a b -> a + b);
-}</pre>
+};</pre>
     </td>
     <td>
       <pre>
 let myFuncs = {
   myFun: fun x => x + 1,
   your: fun a b => a + b
-}</pre>
-    </td>
-  <tr>
-    <td>
-      <pre>
-let myRec = {x = 0; y = 10}</pre>
-    </td>
-    <td>
-      <pre>
-let myRec = {x: 0, y: 10};</pre>
```

Comment:
Because it's a duplicate, see lines [L316](https://github.com/facebook/reason/blob/907c5d3b3ba0011cc9fbedf1372dbb38e62c23c9/mlCompared.html#L316) and [L341](https://github.com/facebook/reason/blob/907c5d3b3ba0011cc9fbedf1372dbb38e62c23c9/mlCompared.html#L341).